gerrit-entrypoint.sh

```diff
@@ -83,13 +83,18 @@ EOF
 done
 
   #Customize gerrit.config
+  [ -z "${MAKE_FIRST_USER_ADMIN}" ] || set_gerrit_config capability.makeFirstUserAdmin "${MAKE_FIRST_USER_ADMIN}"
+  [ -z "${ENABLE_SIGNED_PUSH}" ] || set_gerrit_config receive.enableSignedPush "${ENABLE_SIGNED_PUSH}"
```

Comment:
This option is existed in both 2.14.x and 2.15.x.
I'd like to merge this kind of options back to 2.14.x branch. So a separated commit would be welcome.